doc/classes/Color.xml

```diff
@@ -492,6 +495,7 @@
 	<members>
 		<member name="a" type="float" setter="" getter="" default="1.0">
 			The color's alpha component, typically on the range of 0 to 1. A value of 0 means that the color is fully transparent. A value of 1 means that the color is fully opaque.
+			[b]Note:[/b] The alpha channel is always stored with linear encoding, regardless of the color space of the other color channels.
```

Comment:
Should probably mention the two methods affected by this, instead, because the note by itself doesn't make sense otherwise. "always stored with linear encoding", but **Color** cannot know its own color space.